apm-agent-core/src/test/java/org/example/stacktrace/ErrorCaptureTest.java

```diff
@@ -68,7 +68,7 @@ void testCulprit() {
     @Test
     void testUnnestNestedException() {
         final NestedException nestedException = new NestedException(new CustomException());
-        ErrorCapture errorCapture = tracer.captureException(nestedException, null, null);
+        ErrorCapture errorCapture = tracer.captureException(nestedException, tracer.currentContext(), null);
```

Comment:
[question] was there any particular reason this becomes required ? The argument remains still `@Nullable` in `ElasticApmTracer#captureException`.

Reply:
I actually forgot to remove the `@Nullable` on that method overload.
`ElasticContext` is supposed to never be null to make life easier. The empty context is used in such cases instead.